tracer/build/crank/os.profiles.yml

```diff
@@ -48,6 +48,7 @@ profiles:
           DD_TRACE_DEBUG: 0
           DD_INTERNAL_TELEMETRY_V2_ENABLED: "{{ telemetry_v2 }}"
           DD_TELEMETRY_METRICS_ENABLED: "{{ enable_metrics }}"
+          DD_TRACE_LOG_DIRECTORY: "{{ linuxProfilerPath }}/{{ commit_hash }}/logs"
```

Comment:
this would end up beeing collected by the local cron task:
0 4 * * * find /home/ubuntu/tracer/* -type d -ctime +1 | xargs rm -rf

as they are all in these folders, that are regularly removed:
![image](https://github.com/DataDog/dd-trace-dotnet/assets/8353486/e41d9c31-8f86-4ae6-9ecc-8ba786033703)

Reply:
so it would still gives us a few days to have a look if something is wrong